src/BrowscapUpdater.php

```diff
@@ -121,22 +120,16 @@ public function setCache($cache)
                 Exception::CACHE_INCOMPATIBLE
             );
         }
-
-        return $this;
```

Comment:
Ah, this breaks BC unnecessarily, please revert; I only meant to not use fluent on new features